frontend/src/app/modules/global_search/tabs/global-search-tabs.component.ts

```diff
@@ -26,20 +26,17 @@
 // See docs/COPYRIGHT.rdoc for more details.
 // ++
 
-import {
-  Component,
-  OnDestroy,
-} from '@angular/core';
+import {Component, OnDestroy} from '@angular/core';
 import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
 import {GlobalSearchService} from "core-app/modules/global_search/services/global-search.service";
 import {Subscription} from "rxjs";
-import {ScrollableTabsComponent} from "core-app/modules/common/tabs/scrollable-tabs.component";
+import {ScrollableTabsComponent} from "core-app/modules/common/tabs/scrollable-tabs/scrollable-tabs.component";
 
 export const globalSearchTabsSelector = 'global-search-tabs';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/565520ae6f2c58886c7e18d555597f7b982aed74/frontend/src/app/modules/global_search/tabs/global-search-tabs.component.ts#L36"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>